### PR TITLE
Environment_Engine: Removed SplitPanelByGeometry due to lack of use and confusing UX

### LINKED ISSUE #3125: Environemnt_Engine Removing or renaming SplitPanelByGeometry.cs
#### What is missing/incorrect?

After a chat with @FraserGreenroyd and looking into analytics SplitPanelByGeometry.cs method is not used (8 uses in total). Furthermore the method can be easily confused when searching for other split methods so it would be good to remove it.

### PR DESCRIPTION
**Pull request by albinber**
### Issues addressed by this PR

Closes #3125

### CODE REVIEW

File: Environment_Engine/Versioning_63.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+
+  "MessageForDeleted": {
+    "BH.Engine.Environmnet.Modify.SplitPanelsByGeometry(BH.oM.Environment.Elements.Panel, System.Collections.Generic.List<BH.oM.Geometry.Polyline>)": "Method has been removed due to lack of use and confusing UX"
```
Comment (FraserGreenroyd) [inline Environment_Engine/Versioning_63.json:4]:
```suggestion
    "BH.Engine.Environment.Modify.SplitPanelsByGeometry(BH.oM.Environment.Elements.Panel, System.Collections.Generic.List<BH.oM.Geometry.Polyline>)": "Method has been removed due to lack of use and confusing UX"
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
My bad @albinber as I gave you the versioning key offline! My poor typing getting in the way this morning 🤕

**Comment by albinber** [comment]
@bhombot check compliance

File: Environment_Engine/Versioning_63.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+
+  "MessageForDeleted": {
+    "BH.Engine.Environment.Modify.SplitPanelsByGeometry(BH.oM.Environment.Elements.Panel, System.Collections.Generic.List<BH.oM.Geometry.Polyline>)": "Method has been removed due to lack of use and confusing UX"
```
Comment (albinber) [inline Environment_Engine/Versioning_63.json:4]:
```suggestion
    "BH.Engine.Environment.Modify.SplitPanelByGeometry(BH.oM.Environment.Elements.Panel, System.Collections.Generic.List<BH.oM.Geometry.Polyline>)": "Method has been removed due to lack of use and confusing UX"
```

**Comment by albinber** [comment]
@bhombot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core
@BHoMBot check serialisation
@BHoMBot check null-handling
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Review by FraserGreenroyd — APPROVED** [review]
Method deleted, versioning happy, good to go 😄

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: unit-tests

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**